### PR TITLE
Feature/xivy 4422 supress notification UI

### PR DESCRIPTION
**Pull request by ivy-fhe**
Added checkbox "Suppress Notification" to TaskView 
Extended tests to account for new checkbox

### CODE REVIEW

File: packages/protocol/src/data/inscription.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export interface Inscription {
   inscriptionRequest: InscriptionRequest;
   inscriptionSaveRequest: InscriptionSaveRequest;
   inscriptionValidation: InscriptionValidation[];
+  javaType: JavaType[];
```
Comment (weissreto) [inline packages/protocol/src/data/inscription.ts:45]: This seems to be coming from another story. May lead to merge conflict with Elio. 
The faster of you two will be happy.

File: packages/protocol/src/data/inscription.ts [hunk 2]
```diff
@@ -55,6 +56,7 @@ export interface Inscription {
   schemaKey: SchemaKey;
   scriptingDataArgs: ScriptingDataArgs;
   string: string;
+  typeSearchRequest: TypeSearchRequest;
```
Comment (weissreto) [inline packages/protocol/src/data/inscription.ts:59]: This seems to be coming from another story. May lead to merge conflict with Elio. 
The faster of you two will be happy.

**Review by weissreto — APPROVED** [review]
Good for me. 
@ivy-lli can you also have a look

File: packages/editor/src/components/parts/task/options/TaskListOptions.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ const TaskListPart = () => {
   return (
     <PathCollapsible label='Options' defaultOpen={task.skipTasklist} path='delay'>
       <Checkbox label='Skip Tasklist' value={task.skipTasklist} onChange={change => update('skipTasklist', change)} />
+      <Checkbox label='Suppress Notification' value={task.notification.suppress} onChange={change => update('notification', {suppress: change})} />
```
Comment (ivy-lli) [inline packages/editor/src/components/parts/task/options/TaskListOptions.tsx:13]: just as info: this works as long as you don't have another property in the notification object
But ok for me

**Review by ivy-lli — APPROVED** [review]
Looks good

**Pull request merged**